app/models/service_template_transformation_plan_task.rb

```diff
@@ -322,4 +350,8 @@ def conversion_options_destination_provider_openstack(cluster, storage)
       :osp_security_groups_ids    => [destination_security_group.ems_ref]
     }
   end
+
+  def valid_states
+    super + %w(migrate)
```

Comment:
If you're just appending a single item you can use the shovel operator so you don't need to wrap it in an array, `super << "migrate"`